src/app/collective-rewards/utils/removeBrackets.ts

```diff
@@ -0,0 +1,10 @@
+export const removeBrackets = (name?: string) => {
+  if (!name) {
+    return ''
+  }
+  const bracketIndex = name.indexOf(']')
+  if (bracketIndex !== -1) {
+    return name.slice(bracketIndex + 1).trimStart()
+  }
+  return name
```

Comment:
```suggestion
  
  return name.replace(/^[^\]]*\]\s*/, '');
```

Reply:
@jurajpiar thank you for the suggestion, but I've avoided the regexp on purpose since I find them less readable and with more difficulty to debug.